pkg/network/ebpf/c/http-maps.h

```diff
@@ -10,7 +10,7 @@ struct bpf_map_def SEC("maps/http_in_flight") http_in_flight = {
     .type = BPF_MAP_TYPE_HASH,
     .key_size = sizeof(conn_tuple_t),
     .value_size = sizeof(http_transaction_t),
-    .max_entries = 0, // This will get overridden at runtime using max_tracked_connections
+    .max_entries = 1, // This will get overridden at runtime using max_tracked_connections
```

Comment:
I think we had these as `0`, so that if for some reason the config value wasn't applied, it would fail rather than start with only one entry.

Reply:
Oh, actually I should clarify why I'm using 1 here on purpose.
Unfortunately the current behavior of the `ebpf.Manager` is to create all maps present in the object file (I already validated that `ExcludedSections` won't help here either). So if I leave this as `0`, I'll have to configure this map size for every SSL/crypto "subprogram", since 0 is an invalid map size

By the way, I'm planning to replace the `ebpf.Manager` stuff by direct calls to the eBPF library in a follow-up PR (for version 7.31) to address this and some other issues we talked about before, and then I should be able to revert this to 0

Reply:
👍